docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -19,7 +19,7 @@ Model Builder only accepts TSV, CSV, and TXT files with comma, tab, and semi-col
 Model Builder helps you create models for the following machine learning scenarios:
 
 - Sentiment analysis (binary classification): Classify textual data into two categories.
```

Comment:
@jwood803 correct me if I'm wrong but I'm not sure if Sentiment Analysis is an option. In which case we might want to remove it from this list.